lib/proxy/forward.c

```diff
@@ -484,6 +487,46 @@ static int forward_cmd_parse_dst(pool *p, const char *arg, char **name,
   return 0;
 }
 
+static int forward_cmd_sni_dst(pool *p, const struct proxy_conn **pconn) {
+
+  const char *default_port = NULL;
+  char * host = NULL, *hostport = NULL, *uri = NULL;
+
+  host = pr_env_get(p, "TLS_SERVER_NAME");
+  if (host == NULL) {
```

Comment:
Do you think there's any value in using the existing, non-SNI behavior, if SNI is ever missing/lacking unexpectedly?